spec/takeaway_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new }
+  let(:dish) { double :dish => "Dish 1"}
+  let(:order) { double :order, inject: [["Dish 1" => 2.00]], length: 3}
+  let(:quantity) { double :quantity => 3}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)